apps/zbugs/server/push-handler.ts

```diff
@@ -1,14 +1,19 @@
-import {PushProcessor} from '@rocicorp/zero/pg';
+import {
+  PushProcessor,
+  ZQLDatabaseProvider,
+  ZQLPostgresJSAdapter,
+} from '@rocicorp/zero/pg';
 import postgres from 'postgres';
 import {schema} from '../shared/schema.ts';
 import {createServerMutators, type PostCommitTask} from './server-mutators.ts';
 import type {AuthData} from '../shared/auth.ts';
-import {connectionProvider} from '@rocicorp/zero/pg';
 import type {ReadonlyJSONValue} from '@rocicorp/zero';
 
 const processor = new PushProcessor(
-  schema,
-  connectionProvider(postgres(process.env.ZERO_UPSTREAM_DB as string)),
+  new ZQLDatabaseProvider(
+    new ZQLPostgresJSAdapter(postgres(process.env.ZERO_UPSTREAM_DB as string)),
```

Comment:
This looks good. We have a few users that use `pg.js` instead of `postgres.js` so this keeps things simple for them.